emcache/client.py

```diff
@@ -623,6 +623,30 @@ async def flush_all(
 
         return
 
+    async def version(self, memcached_host_address: MemcachedHostAddress) -> Optional[Item]:
```

Comment:
I would go for returning just an `Optional[str]`, `Item` data type is reserved for key values.

Reply:
Of course, thanks, we'll redo it too)

Reply:
+